packages/icons/general/LinkOut.tsx

```diff
@@ -0,0 +1,40 @@
+import { createPaletteAwareIcon } from '../utils'
+
+export const LinkOutIcon = createPaletteAwareIcon(
+    'LinkOutIcon',
+    <>
+        <path
```

Comment:
plz use https://aka.pw/svgomg reduce svg size

Reply:
It has been reduced